src/client/protocolgameparse.cpp

```diff
@@ -1122,16 +1162,24 @@ void ProtocolGame::parseDistanceMissile(const InputMessagePtr& msg)
     g_map.addThing(missile, fromPos);
 }
 
-void ProtocolGame::parseCreatureMark(const InputMessagePtr& msg)
+void ProtocolGame::parseItemClasses(const InputMessagePtr& msg)
 {
-    const uint id = msg->getU32();
-    const int color = msg->getU8();
+    int classSize = msg->getU8();
+    int tiersSize = 0;
+    for(uint8_t i = 0; i < classSize; i++) {
+        msg->getU8(); // class id
 
-    const CreaturePtr creature = g_map.getCreatureById(id);
-    if(creature)
-        creature->addTimedSquare(color);
-    else
-        g_logger.traceError("could not get creature");
+        // tiers
+        tiersSize = msg->getU8();
+        for(uint8_t j = 0; j < tiersSize; j++) {
+            msg->getU8(); // tier id
+            msg->getU64(); // upgrade cost
+        }
+    }
+
+    for(uint8_t i = 0; i < tiersSize + 1; i++) {
+        msg->getU8(); // ??
+    }
```

Comment:
I have a problem with this changes...
My client 772 was using this this opcode (134) to draw the black square on an attacking creature.
I fixed my personal code by adding a condition to check my server version.
I'm now sure in which protocol this opcode was replaced from "creatureMark" to "itemClasses"..

Reply:
item classes is feature from tibia 1281, its very recent

Reply:
Done, thanks